test/e2e/scenario/rum/resources.scenario.ts

```diff
@@ -196,6 +196,50 @@ describe('rum resources', () => {
       })
   })
 
+  createTest('track redirect fetch timings')
+    .withRum()
+    .run(async ({ serverEvents }) => {
+      await browserExecuteAsync((done) => {
+        fetch(`/redirect?duration=${200}`).then(
+          () => done(undefined),
+          () => {
+            throw Error('Issue with fetch call')
+          }
+        )
+      })
+      await flushEvents()
+      const resourceEvent = serverEvents.rumResources.find((r) => r.resource.url.includes('/redirect'))!
+      expect(resourceEvent).not.toBeUndefined()
+      expect(resourceEvent.resource.method).toEqual('GET')
+      expect(resourceEvent.resource.status_code).toEqual(200)
+      expect(resourceEvent._dd.resolveDuration).toBeGreaterThan(0)
+      expectToHaveValidTimings(resourceEvent)
+      expect(resourceEvent.resource.redirect).not.toBeUndefined()
+      expect(resourceEvent.resource.redirect!.duration).toBeGreaterThan(0)
+    })
+
+  createTest('track dns lookup fetch timings')
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ ‏this test is an exact copy of the one before. Typo?